doc/Doxyfile

```diff
@@ -791,7 +791,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ../src
```

Comment:
I believe this wrong and the commit should be removed

Reply:
`../src` is indeed the location of the source directory relative to the doxyfile. The fix for this pr should be to run doxygen from inside the doc directory, I think.

Reply:
I agree, updated.